### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by alex83we**
Translation of the de.json file maybe you can take over the version adapted for Laravel 8 (Laravel / Jetstream)

### CODE REVIEW

File: json/de.json [hunk 1]
```diff
@@ -1,75 +1,164 @@
...
+    "If necessary, you may logout of all of your other browser sessions across all of your devices. If you feel your account has been compromised, you should also update your password.": "Bei Bedarf können Sie sich von allen anderen Browsersitzungen auf allen Ihren Geräten abmelden. Wenn Sie der Meinung sind, dass Ihr Konto kompromittiert wurde, sollten Sie auch Ihr Passwort aktualisieren.",
+    "If you did not create an account, no further action is required.": "Wenn Sie kein Konto erstellt haben, sind keine weiteren Maßnahmen erforderlich.",
+    "If you did not receive the email": "Wenn Sie die E-Mail nicht erhalten haben",
+    "If you did not request a password reset, no further action is required.": "Wenn Sie kein Zurücksetzen des Kennworts angefordert haben, sind keine weiteren Maßnahmen erforderlich.",
+    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Wenn Sie Probleme beim Klicken auf die Schaltfläche \": actionText\" haben, kopieren Sie die folgende URL und fügen Sie sie in Ihren Webbrowser:",
```
Comment (caouecs) [inline json/de.json:58]: Problem on the second :actionText

**Review by caouecs — CHANGES_REQUESTED** [review]
Danke but there is a typo

**Comment by alex83we** [comment]
Where is the typo in your opinion?

Fred Delrieu <notifications@github.com> schrieb am Fr., 25. Sept. 2020,
15:28:

File: json/de.json (hunk 1 above)
Comment (caouecs) [reply json/de.json:58]: you have added un space between : and actionText

**Pull request closed without merging**